engine/tests/ElggUserTest.php

```diff
@@ -177,6 +179,9 @@ public function testElggUserRemoveAdmin() {
 
 		$this->assertTrue($this->user->removeAdmin());
 
+		// need to save user to propagate attributes to the database
+		$this->user->save();
```

Comment:
Never mind. I broke something. I thought it was just setting the attribute without saving the entity.